micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/IsGraphitePublishingEnabled.groovy

```diff
@@ -0,0 +1,26 @@
+package com.ofg.infrastructure.metrics.config
+
+import org.springframework.context.annotation.Condition
+import org.springframework.context.annotation.ConditionContext
+import org.springframework.core.type.AnnotatedTypeMetadata
+
+class IsGraphitePublishingEnabled implements Condition {
+
+    private static final Set<String> ENABLING_VALUES = ['on', 'true', 'yes']
+    public static final String GRAPHITE_PUBLISHING = 'graphite.publishing.enabled'
```

Comment:
Great, it should be consistent with: https://github.com/4finance/micro-infra-spring/blob/72981e8edc392ee897845fa250c353b34692c4ca/micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/metrics/config/MetricsConfiguration.groovy#L51